doc/src/build/install.md

```diff
@@ -174,6 +174,8 @@ In macOS, other than the aforementioned [Brew](#brew) package manager, the gener
 
 In Microsoft Windows, also install:
 
+[Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/)
```

Comment:
we should mention that this is only applicable to Windows on ARM64

Reply:
> we should mention that this is only applicable to Windows on ARM64

Done. Thanks for the reminder!